docs/approach.rst

```diff
@@ -165,24 +183,18 @@ These components are subjected to component selection, the specifics of which
 vary according to algorithm.
 
 In the simplest approach, ``tedana`` uses Minka’s MLE to estimate the
-dimensionality of the data, which disregards low-variance components.
+dimensionality of the data, which disregards low-variance components. (the `mle` option in for `--tedpca`).
 
-A more complicated approach involves applying a decision tree to identify and
+A more complicated approach involves applying a decision tree (similar to the
```

Comment:
Is this more complicated, really ? I'd argue it's simpler :laughing: 

More seriously, can we cite the PNAS paper here where they describe this in depth ?

Reply:
yuppers. cited